src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -348,7 +346,7 @@ protected void onPreExecute() {
           protected Void doInBackground(Void... params) {
             for (long chatId : selectedConversations) {
               if (chatId == DcChat.DC_CHAT_ID_DEADDROP) {
-                dcContext.marknoticedContact(getListAdapter().getDeaddropContactId());
+                dcContext.marknoticedChat(DcChat.DC_CHAT_ID_DEADDROP);
```

Comment:
same here: `dc_marknoticed_chat(DC_CHAT_ID_DEADDROP)` won't work, it would just be a no-op as core the [corresponding sql-statement](https://github.com/deltachat/deltachat-core-rust/blob/master/src/chat.rs#L1944) works with real-chat-ids.

i think, the idea of this handling was to "move the deaddrop away" when the user taps "delete" for it. while archiving above is already a bit weird, moving the deaddrop only "out of sight" when the user selects "delete" seems to be wrong to me.